pkg/controller/common/helm/manifestprocessing.go

```diff
@@ -73,10 +73,14 @@ func (p *ManifestProcessor) ProcessManifest(ctx context.Context, man manifest.Ma
 			allErrors = append(allErrors, err)
 			continue
 		}
+		if len(rawJSON) == 0 || string(rawJSON) == "{}" || string(rawJSON) == "null" {
+			// MAISTRA-2227 ignore empty objects.  this could happen if charts have in empty blocks, e.g. superfluous "---"
+			continue
+		}
 		obj := &unstructured.Unstructured{}
 		_, _, err = unstructured.UnstructuredJSONScheme.Decode(rawJSON, nil, obj)
 		if err != nil {
-			log.Error(err, "unable to decode object into Unstructured")
+			log.Error(err, fmt.Sprintf("unable to decode object into Unstructured:\n%s\n", raw))
```

Comment:
is there any chance we'd log something we don't intend to here?

Reply:
I think it's ok. I think secrets and similar would be in other resources.

Reply:
okay, made this more like a debug entry, leaving the error as it was